Add glt for compressing and searching unstructured logs using an optimized storage structure. (#237)
y-scope/clp: components/core/README.md

## Running

* CLP contains four core executables: `clp`, `clg`, `clp-s` and `glt`.
  * `clp` is used for compressing and extracting unstructured (plain text) logs.
  * `clg` is used for performing wildcard searches on the compressed unstructured logs.
  * `clp-s` is used for compressing and searching semi-structured logs (e.g., JSON) with support for
    handling highly dynamic schemas.

>   * `glt` is a version of clp specialized for search, used for compressing and extracting unstructured (plain text) logs.

Reviewer: ```suggestion
  * `glt` is a version of clp specialized for searching unstructured (plain text) logs.
```

> See [Using CLP for unstructured logs](../../docs/core/clp-unstructured.md), 
> [Using CLP for semi-structured logs](../../docs/core/clp-structured.md) and 

Reviewer: ```suggestion
See [Using CLP for unstructured logs](../../docs/core/clp-unstructured.md),
[Using CLP for semi-structured logs](../../docs/core/clp-structured.md) and
```

> [Using GLR for unstructured logs](../../docs/core/glt.md) for usage instructions.

Reviewer: ```suggestion
[Using GLT for unstructured logs](../../docs/core/glt.md) for usage instructions.
```
